bindings/nodejs/lib/types/models/conflict-reason.ts

```diff
@@ -62,3 +62,30 @@ export declare enum ConflictReason {
      */
     semanticValidationFailed = 255,
 }
+
+/**
+ * Conflict reason strings.
+ */
+export const CONFLICT_REASON_STRINGS: { [key in ConflictReason]: string } = {
```

Comment:
Not sure about this, it used to be in there, but sounds ike more of a front end logic thing

Reply:
Agree. Could be we need a new util method to get the description?